src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -522,7 +526,11 @@ def _unpack_build(self,
         if not self._unpack_everything:
           # We will never unpack the full build so we need to get the targets
           # from the build archive.
+          list_fuzz_target_start_time = time.time()
           self._fuzz_targets = list(build.list_fuzz_targets())
+          _emit_job_build_retrieval_metric(list_fuzz_target_start_time,
```

Comment:
You also need to emit this metric in the `fuzz_targets` property method. Meant to comment there, but it looks like I can't.